packages/dashboard/src/customization/widgets/table/emptyTableState.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, Button, SpaceBetween } from '@cloudscape-design/components';
+import type { FunctionComponent } from 'react';
+
+const EmptyTableState: FunctionComponent = () => {
+  return (
+    <Box data-testid='emptyStateTableDisplay' margin={{ vertical: 'xs' }} textAlign='center' color='inherit'>
+      <SpaceBetween size='m'>
+        <b data-testid='default-msg'>No data to display</b>
+        <Button>Add asset properties</Button>
```

Comment:
i guess, we discussed about this, no button required here

Reply:
I will confirm with Navita and will update accordingly

Reply:
Resolved